lib/ui/window/viewport_metrics.cc

```diff
@@ -40,8 +40,8 @@ ViewportMetrics::ViewportMetrics(double p_device_pixel_ratio,
           p_physical_system_gesture_inset_bottom),
       physical_system_gesture_inset_left(p_physical_system_gesture_inset_left) {
   // Ensure we don't have nonsensical dimensions.
-  FML_DCHECK(physical_width > 0);
-  FML_DCHECK(physical_height > 0);
+  FML_DCHECK(physical_width >= 0);
```

Comment:
Is this related to jit_release mode?

Reply:
Yes, see https://github.com/flutter/engine/pull/12446#discussion_r328337326

Reply:
I think these checks are just too strong and it should be possible (and often is on release builds) that the viewport metrics dimensions can be zero. Let's just get rid of them.

Reply:
I've removed all of them